build-system/tasks/lint.js

```diff
@@ -127,7 +127,7 @@ function lint() {
   if (argv.fix) {
     options.fix = true;
   }
-  const stream = initializeStream(config.lintGlobs, {});
+  const stream = initializeStream(argv.file || config.lintGlobs, {});
```

Comment:
@bradfrizzell Sorry about not bringing this up earlier, but I just realized that it makes sense to change this from `argv.file` to `argv.files`, and allow developers to lint more than one file that they're editing in a branch, by specifying a list of files or a glob.

Reply:
done